### ISSUE TITLE
QUESTION POLL:: Matching

### ISSUE THREAD

**Issue by pawelczak** [pawelczak/EasyAutocomplete#153]
Default value of matching is disabled. I'm thinking on changing it, so it matching would be enabled by default. I wonder if it is a good idea. I would make a poll on this by github has no such feature, so maybe if anyone has an opinion on this matter, please write in comments.

**Comment by thisismyaim** [comment]
yes you could enable by default but the project i am working on i used easyautocomplete and got problem while using [ match enabled true ] it matches with first match word like if i type home decoration it will match but if i type i want home decoration it won't match. That's why i am using highlight query in elasticsearch to match word.

**Comment by pawelczak** [comment]
@thisismyaim Did you try to activate sorting feature (http://easyautocomplete.com/guide#sec-sort)?

**Comment by thisismyaim** [comment]
Yes i used this.

**Comment by sileence** [comment]
I do think that using matching enabled by default makes more sense. To be honest I didn't understand at first what `matching = false` meant, because this is an autocomplete plugin. Leave the feature there because it could be useful, but true by default, please :)
